.run/main.dart.run.xml

```diff
@@ -4,4 +4,24 @@
     <option name="filePath" value="$PROJECT_DIR$/admin-frontend/app_singleapp/lib/main.dart" />
     <method v="2" />
   </configuration>
+  <configuration default="false" name="main.dart" type="FlutterRunConfigurationType" factoryName="Flutter" nameIsGenerated="true">
```

Comment:
junk?

Reply:
We have to put this in the .gitignore file. Driving me crazy.

Reply:
still here :)